actions/lib/actions.py

```diff
@@ -104,6 +104,36 @@ def get_node(self, node):
 
         return orion_node
 
+    def get_agent(self, orion_node):
```

Comment:
I would prefer if this function took `npm_id` as a parameter, instead of `orion_node`.